addons/io/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/api/HueState.java

```diff
@@ -30,6 +30,10 @@
     public String colormode = "ct";
     public boolean reachable = true;
 
+    public static short convertBri(int percentBrightness) {
```

Comment:
Can you please spell out `Brightness` here, I don't think that the shorter name adds value :-)

Reply:
I only used the short name bri, because it was the original variable name that gets converted and bri is the name of the json part sending to hue.

But Shure can be renamed.

Reply:
> I only used the short name bri, because it was the original variable name

Yes I already realized that after reviewing the rest, but thanks for changing it.